frontend/src/explorer/explorer-event-controller.ts

```diff
@@ -311,10 +311,15 @@ class ExplorerEventController {
             landing = false;
         }
         const title = `${landing ? 'My' : 'Sample'} ${queryMode}`;
+        const i18nKey = `button.${title.toLowerCase().replace(' ', '-')}`;
         const endpoint = `${queryMode}:${landing ? 'user' : 'sample'}`;
         const collection = new FlatItemCollection(ldChannel.request(endpoint));
         const browsePanel = new SearchResultListPanel({
-            title,
+            // i18next.t('button.my-sources', 'My sources');
+            // i18next.t('button.sample-sources', 'Sample sources');
+            // i18next.t('button.my-items', 'My items');
+            // i18next.t('button.sample-items', 'Sample items');
+            title: i18next.t(i18nKey, title),
```

Comment:
comments should probably go?

Reply:
No, they are needed to ensure that i18next-parser doesn't purge these keys on the next scan.

Reply:
Can you elaborate on this? Why is there a difference in behaviour between commented code and removed code?

Reply:
It's not outcommented. It's a "clarification" (for the parser) of what the next line means.